### ISSUE TITLE
ember-cli-page-object clobbers host app version of ember-test-helpers

### ISSUE THREAD

**Issue by magistrula** [san650/ember-cli-page-object#361]
Observed in an app that uses `ember-test-helpers@0.6.3`, when testing the changes for #357:

`import wait from 'ember-test-helpers/test'` pulls in `wait` from `v0.7.13` (`ember-cli-page-object` version) instead of from `v0.6.3` (host-app version).

**Comment by san650** [comment]
After discussing with @magistrula we came to the conclusion that instead of adding `ember-test-helpers` as a dependency of ember-cli-page-object, we need to move it as a dev-dependency (because of this addon's tests) and install the dependency on the host app using the installation blueprint.

https://ember-cli.com/extending/#working-with-dependencies

This way we'll update host's `ember-test-helpers` dependency if needed.

cc/ @bendemboski

**Comment by bendemboski** [comment]
@san650 @magistrula 

I chatted with some of the Ember core team folks, and they recommend something a little different. The intent is that apps won't specify their own version of `ember-test-helpers` (now called `@ember/test-helpers`), but will just use the version pulled in by `ember-qunit` or `ember-mocha` (which is what your project was doing @magistrula -- apparently correctly, sorry for the bum steer).

So `ember-cli-page-object` really shouldn't have it as a dependency _or_ install it in the host app using the blueprint. If anything, we should be checking for a new enough `ember-qunit` or `ember-mocha` version at runtime, although I don't think that's really necessary unless/until we deprecate pre-RFC268 tests.

So to sum it up, it sounds like we should just simply remove it -- anybody running tests will already have the version that they need installed via `ember-qunit` or `ember-mocha`.

Make sense?
